assignments/this.js

```diff
@@ -1,26 +1,62 @@
 /* The for principles of "this";
 * in your own words. explain the four principle for the "this" keyword below.
 *
-* 1. 
-* 2. 
-* 3. 
-* 4. 
+* 1. window
```

Comment:
Great job identifying the four principles! Can you elaborate and explain when each principle takes effect?